vector/src/main/scala/geotrellis/vector/io/json/JsonFeatureCollection.scala

```diff
@@ -99,11 +99,11 @@ object JsonFeatureCollection{
     fc
   }
 
-  def apply(geometries: Traversable[Geometry]) = {
+/*  def apply(geometries: Traversable[Geometry]) = {
     val fc = new JsonFeatureCollection()
     fc ++= geometries.toList
     fc
-  }
+  }*/
 
```

Comment:
ok, done, also in JsonFeatureCollectionMap. The travis-CI will probably still fail, the sbt update is not in this pull request.